source/java/src/org/lucee/extension/cache/eh/EHCacheSupport.java

```diff
@@ -29,10 +29,19 @@
 
 import org.lucee.extension.cache.CacheSupport;
 import org.lucee.extension.cache.eh.util.TypeUtil;
+import lucee.loader.engine.CFMLEngineFactory;
+import lucee.commons.io.log.Log;
 
 public abstract class EHCacheSupport extends CacheSupport implements Cache {
 	
 	protected boolean isDistributed;
+	protected boolean isSerialized;
+
+	public static Log getLogger() {
```

Comment:
If a different {{config}} object can be passed in, unless I'm missing something, it doesn't make sense to store the logger as a property in the instance (since a different object can be returned). So instead, I'm just caching the log instance returned in the code now, so the getLogger() is only called once.